site/docs/master/sonobuoy-config.md

```diff
@@ -54,14 +54,45 @@ Filters
    - A Kubernetes [label selector][labelselector] which will be added to every query run.
 
 Limits
- - Options for limiting limits the scope of response when getting logs from pods. These will be passed onto Kubernetes [PodLogOptions][podlogopts]
- - The supported parameters are:
-    - Previous
-    - SinceSeconds
-    - SinceTime
-    - Timestamps
-    - TailLines
-    - LimitBytes
+ - Options for limiting limits the scope of response.
+ - **Limits.PodLogs** limits the scope when getting logs from pods. 
+     - The scope of pods is defined by:
```

Comment:
Slightly confusing; I'd be a little more verbose to make it clear (I know you spell out each field below, but this is the first thing they'll see and shows how those fields relate). Maybe even just remove this section to clarify down below in each field (going to put a few suggestions there)

Reply:
understood